template/config/cypress/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "scripts": {
-    "test:e2e": "start-server-and-test preview http://127.0.0.1:4173/ 'cypress open'",
-    "test:e2e:ci": "start-server-and-test preview http://127.0.0.1:4173/ 'cypress run'"
+    "test:e2e": "start-server-and-test preview http://127.0.0.1:4173/ 'cypress open --e2e'",
```

Comment:
Currently `--e2e` and `--component` flags are always passed to skip the "Choose testing type" screen.
This is because the commands are `test:e2e` and `test:unit`, we expect users to know what types of tests they are looking for.